skypy/pipeline/tests/test_driver.py

```diff
@@ -41,6 +41,17 @@ def test_driver():
     with fits.open('test_table.fits') as hdu:
         assert np.all(Table(hdu[1].data) == driver.test_table)
 
+    # Check for failure if output files already exist and overwrite is False
+    driver = SkyPyDriver()
+    with pytest.raises(OSError):
+        driver.execute(config, file_format='fits', overwrite=False)
+
+    # Check that the existing output files are modified if overwrite is True
+    timestamp = os.path.getmtime("test_table.fits")
+    driver = SkyPyDriver()
+    driver.execute(config, file_format='fits', overwrite=True)
+    assert(os.path.getmtime("test_table.fits") > timestamp)
```

Comment:
This might not have the time resolution if `getmtime` return seconds since the epoch? Waiting explicitly might help.

Reply:
Or, easier, make timestamp now and check that mtime >= now

Reply:
I get sub-second precision, but that could be implementation/OS dependent

Reply:
Yeah the python docs only say 

> os.path.getmtime(path)
Return the time of last modification of path. The return value is a number giving the number of seconds since the epoch (see the time module). Raise os.error if the file does not exist or is inaccessible.

so I am not sure what we can infer from that.

Reply:
Instead I now change `size` in the config file and check that the table in the file has the correct length i.e. it has been overwritten with new data